dutch/bos/ontvoering/meehelpen/klus/geld-verdienen.md

```diff
@@ -2,4 +2,8 @@ In eerste instantie geloven de mannen je niet maar als ze je Rolex om je arm
 zien zijn ze snel geïnteresseerd. Wie is deze man?
 
 De mannen kijken elkaar aan en na een korte blik vraagt de langste man:
-Voor wie werk jij dan eigenlijk en heb je nog meer klussen?
\ No newline at end of file
+Voor wie werk jij dan eigenlijk en heb je nog meer klussen?
+
+ja, zeg je, en glimlacht duister. Ik bel mijn baas wel even of hij nog wat voor 
+jullie in petto heeft. Je pakt je telefoon, maar je belt helemaal niet je baas,
+je bent bezig de [politie te bellen](politie/politie.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.